nix/buffrs.nix

```diff
@@ -37,5 +37,16 @@ in {
 
     # Audit licenses
     buffrs-deny = craneLib.cargoDeny { inherit src; };
+
+    # Rust unit and integration tests
+    buffers-nextest = craneLib.cargoNextest (commonArgs // {
+      inherit cargoArtifacts;
+      partitions = 1;
+      partitionType = "count";
+      # Ignore tutorial and publish tests because they need TLS certificates 
+      # which doesn't yet work in our nix setup
```

Comment:
Lets not ignore them, you can likely add the following to the `cargoNexttest` args:

```
SSL_CERT_FILE = "${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt";
```

Reply:
This will prob make there cert available to the tests

Reply:
Ooooh! I didn't realize you can just export env vars in there. That's so much easier!